json/src/spec/instant_seal.rs

```diff
@@ -0,0 +1,32 @@
+// Copyright 2015-2018 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Instant seal engine params deserialization.
+
+/// Instant seal engine params deserialization.
+#[derive(Debug, PartialEq, Deserialize)]
+pub struct InstantSealParams {
+	/// Whether to enable millisecond timestamp.
+	#[serde(rename="millisecondTimestamp")]
+	pub millisecond_timestamp: Option<bool>,
```

Comment:
Maybe use `#[serde(default)]` and make it a `bool`? `Option<bool>` seems like an overkill.